[2022-09-10 00:00:01] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 00:00:04] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 00:00:07] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 00:00:10] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 4 次重试 
[2022-09-10 00:00:13] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 5 次重试 
[2022-09-10 00:00:16] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 6 次重试 
[2022-09-10 00:00:20] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 7 次重试 
[2022-09-10 00:00:23] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 8 次重试 
[2022-09-10 00:00:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 9 次重试 
[2022-09-10 00:00:29] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 10 次重试 
[2022-09-10 00:00:32] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1662739199,"self_id":3515950251,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":88,"packet_sent":77,"packet_lost":0,"message_received":3,"message_sent":1,"last_message_time":1662738908,"disconnect_times":0,"lost_times":0}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2022-09-10 00:00:34] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 00:00:37] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 00:00:40] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 00:00:43] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 4 次重试 
[2022-09-10 00:00:46] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 5 次重试 
[2022-09-10 00:00:49] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 6 次重试 
[2022-09-10 00:00:52] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 7 次重试 
[2022-09-10 00:00:55] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 8 次重试 
[2022-09-10 00:00:58] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 9 次重试 
[2022-09-10 00:01:01] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 10 次重试 
[2022-09-10 00:01:04] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1662739232,"self_id":3515950251,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":89,"packet_sent":78,"packet_lost":0,"message_received":3,"message_sent":1,"last_message_time":1662738908,"disconnect_times":0,"lost_times":0}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2022-09-10 00:01:06] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 00:01:09] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 00:01:13] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 00:01:16] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 4 次重试 
[2022-09-10 00:01:19] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 5 次重试 
[2022-09-10 00:01:22] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 6 次重试 
[2022-09-10 00:01:25] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 7 次重试 
[2022-09-10 00:01:28] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 8 次重试 
[2022-09-10 00:01:31] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 9 次重试 
[2022-09-10 00:01:34] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 10 次重试 
[2022-09-10 00:01:37] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1662739264,"self_id":3515950251,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":90,"packet_sent":79,"packet_lost":0,"message_received":3,"message_sent":1,"last_message_time":1662738908,"disconnect_times":0,"lost_times":0}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2022-09-10 00:01:39] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 00:01:42] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 00:01:43] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 00:01:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 00:01:46] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 00:01:48] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 4 次重试 
[2022-09-10 00:01:49] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 00:01:51] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 5 次重试 
[2022-09-10 00:01:52] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 4 次重试 
[2022-09-10 00:01:54] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 6 次重试 
[2022-09-10 00:01:55] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 5 次重试 
[2022-09-10 00:01:57] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 7 次重试 
[2022-09-10 00:01:58] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 6 次重试 
[2022-09-10 00:02:00] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 8 次重试 
[2022-09-10 00:02:01] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 7 次重试 
[2022-09-10 00:02:03] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 9 次重试 
[2022-09-10 00:02:04] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 8 次重试 
[2022-09-10 00:02:06] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 10 次重试 
[2022-09-10 00:02:07] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 9 次重试 
[2022-09-10 00:02:09] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1662739297,"self_id":3515950251,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":91,"packet_sent":80,"packet_lost":0,"message_received":3,"message_sent":1,"last_message_time":1662738908,"disconnect_times":0,"lost_times":0}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2022-09-10 00:02:10] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 10 次重试 
[2022-09-10 00:02:11] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 00:02:14] [WARNING]: 上报 Event 数据 {"post_type":"notice","notice_type":"group_increase","time":1662739301,"self_id":3515950251,"sub_type":"approve","group_id":550305095,"operator_id":0,"user_id":2372822773}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2022-09-10 00:02:15] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 00:02:16] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 00:02:18] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 00:02:19] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 00:02:21] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 4 次重试 
[2022-09-10 00:02:22] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 00:02:24] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 5 次重试 
[2022-09-10 00:02:25] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 4 次重试 
[2022-09-10 00:02:27] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 6 次重试 
[2022-09-10 00:02:28] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 5 次重试 
[2022-09-10 00:02:30] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 7 次重试 
[2022-09-10 00:02:31] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 6 次重试 
[2022-09-10 00:02:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 8 次重试 
[2022-09-10 00:02:34] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 7 次重试 
[2022-09-10 00:02:36] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 9 次重试 
[2022-09-10 00:02:37] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 8 次重试 
[2022-09-10 00:02:39] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 10 次重试 
[2022-09-10 00:02:40] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 9 次重试 
[2022-09-10 00:02:42] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1662739329,"self_id":3515950251,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":105,"packet_sent":91,"packet_lost":0,"message_received":3,"message_sent":1,"last_message_time":1662738908,"disconnect_times":0,"lost_times":0}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2022-09-10 00:02:43] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 10 次重试 
[2022-09-10 00:02:44] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 00:02:46] [WARNING]: 上报 Event 数据 {"post_type":"notice","notice_type":"group_increase","time":1662739334,"self_id":3515950251,"sub_type":"approve","user_id":1433116981,"group_id":550305095,"operator_id":0}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2022-09-10 00:02:47] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 00:02:48] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 00:02:50] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 00:02:51] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 00:02:53] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 4 次重试 
[2022-09-10 00:02:54] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 00:02:56] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 5 次重试 
[2022-09-10 00:02:58] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 4 次重试 
[2022-09-10 00:02:59] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 6 次重试 
[2022-09-10 00:03:01] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 5 次重试 
[2022-09-10 00:03:02] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 7 次重试 
[2022-09-10 00:03:04] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 6 次重试 
[2022-09-10 00:03:05] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 8 次重试 
[2022-09-10 00:03:07] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 7 次重试 
[2022-09-10 00:03:08] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 9 次重试 
[2022-09-10 00:03:10] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 8 次重试 
[2022-09-10 00:03:11] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 10 次重试 
[2022-09-10 00:03:13] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 9 次重试 
[2022-09-10 00:03:15] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1662739362,"self_id":3515950251,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":109,"packet_sent":95,"packet_lost":0,"message_received":3,"message_sent":1,"last_message_time":1662738908,"disconnect_times":0,"lost_times":0}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2022-09-10 00:03:16] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 10 次重试 
[2022-09-10 00:03:17] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 00:03:19] [WARNING]: 上报 Event 数据 {"post_type":"notice","notice_type":"group_increase","time":1662739366,"self_id":3515950251,"sub_type":"approve","operator_id":0,"user_id":1164800221,"group_id":550305095}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2022-09-10 00:03:20] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 00:03:23] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 00:03:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 4 次重试 
[2022-09-10 00:03:29] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 5 次重试 
[2022-09-10 00:03:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 6 次重试 
[2022-09-10 00:03:35] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 7 次重试 
[2022-09-10 00:03:38] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 8 次重试 
[2022-09-10 00:03:41] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 9 次重试 
[2022-09-10 00:03:44] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 10 次重试 
[2022-09-10 00:03:47] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1662739395,"self_id":3515950251,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":112,"packet_sent":98,"packet_lost":0,"message_received":3,"message_sent":1,"last_message_time":1662738908,"disconnect_times":0,"lost_times":0}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2022-09-10 00:03:49] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 00:03:52] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 00:03:55] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 00:03:58] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 4 次重试 
[2022-09-10 00:04:01] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 5 次重试 
[2022-09-10 00:04:04] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 6 次重试 
[2022-09-10 00:04:07] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 7 次重试 
[2022-09-10 00:04:11] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 8 次重试 
[2022-09-10 00:04:14] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 9 次重试 
[2022-09-10 00:04:17] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 10 次重试 
[2022-09-10 00:04:20] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1662739427,"self_id":3515950251,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":114,"packet_sent":100,"packet_lost":0,"message_received":3,"message_sent":1,"last_message_time":1662738908,"disconnect_times":0,"lost_times":0}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2022-09-10 00:04:22] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 00:04:25] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 00:04:28] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 00:04:31] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 4 次重试 
[2022-09-10 00:04:34] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 5 次重试 
[2022-09-10 00:04:37] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 6 次重试 
[2022-09-10 00:04:40] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 7 次重试 
[2022-09-10 00:04:43] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 8 次重试 
[2022-09-10 00:04:46] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 9 次重试 
[2022-09-10 00:04:49] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 10 次重试 
[2022-09-10 00:04:52] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1662739460,"self_id":3515950251,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":115,"packet_sent":101,"packet_lost":0,"message_received":3,"message_sent":1,"last_message_time":1662738908,"disconnect_times":0,"lost_times":0}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2022-09-10 00:04:54] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 00:04:57] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 00:05:00] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 00:05:03] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 4 次重试 
[2022-09-10 00:05:06] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 5 次重试 
[2022-09-10 00:05:09] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 6 次重试 
[2022-09-10 00:05:13] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 7 次重试 
[2022-09-10 00:05:16] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 8 次重试 
[2022-09-10 00:05:19] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 9 次重试 
[2022-09-10 00:05:22] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 10 次重试 
[2022-09-10 00:05:25] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1662739492,"self_id":3515950251,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":117,"packet_sent":103,"packet_lost":0,"message_received":3,"message_sent":1,"last_message_time":1662738908,"disconnect_times":0,"lost_times":0}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2022-09-10 00:05:27] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 00:05:30] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 00:05:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 00:05:36] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 4 次重试 
[2022-09-10 00:05:39] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 5 次重试 
[2022-09-10 00:05:42] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 6 次重试 
[2022-09-10 00:05:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 7 次重试 
[2022-09-10 00:05:48] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 8 次重试 
[2022-09-10 00:05:51] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 9 次重试 
[2022-09-10 00:05:54] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 10 次重试 
[2022-09-10 00:05:57] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1662739525,"self_id":3515950251,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":118,"packet_sent":104,"packet_lost":0,"message_received":3,"message_sent":1,"last_message_time":1662738908,"disconnect_times":0,"lost_times":0}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2022-09-10 00:05:59] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 00:06:02] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 00:06:05] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 00:06:09] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 4 次重试 
[2022-09-10 00:06:12] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 5 次重试 
[2022-09-10 00:06:15] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 6 次重试 
[2022-09-10 00:06:18] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 7 次重试 
[2022-09-10 00:06:21] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 8 次重试 
[2022-09-10 00:06:24] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 9 次重试 
[2022-09-10 00:06:27] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 10 次重试 
[2022-09-10 00:06:30] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1662739557,"self_id":3515950251,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":120,"packet_sent":106,"packet_lost":0,"message_received":3,"message_sent":1,"last_message_time":1662738908,"disconnect_times":0,"lost_times":0}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2022-09-10 00:06:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 00:06:35] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 00:06:38] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 00:06:41] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 4 次重试 
[2022-09-10 00:06:44] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 5 次重试 
[2022-09-10 00:06:47] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 6 次重试 
[2022-09-10 00:06:50] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 7 次重试 
[2022-09-10 00:06:53] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 8 次重试 
[2022-09-10 00:06:56] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 9 次重试 
[2022-09-10 00:07:00] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 10 次重试 
[2022-09-10 00:07:03] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1662739590,"self_id":3515950251,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":121,"packet_sent":107,"packet_lost":0,"message_received":3,"message_sent":1,"last_message_time":1662738908,"disconnect_times":0,"lost_times":0}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2022-09-10 00:07:05] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 00:07:08] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 00:07:11] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 00:09:15] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 00:09:18] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 00:09:21] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 00:09:24] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 4 次重试 
[2022-09-10 00:09:27] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 5 次重试 
[2022-09-10 00:09:31] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 6 次重试 
[2022-09-10 00:09:34] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 7 次重试 
[2022-09-10 00:09:37] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 8 次重试 
[2022-09-10 00:09:40] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 9 次重试 
[2022-09-10 00:09:43] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 10 次重试 
[2022-09-10 00:09:46] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1662739753,"self_id":3515950251,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":135,"packet_sent":121,"packet_lost":0,"message_received":3,"message_sent":2,"last_message_time":1662738908,"disconnect_times":0,"lost_times":0}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2022-09-10 00:09:48] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 00:09:51] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 00:09:54] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 00:09:57] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 4 次重试 
[2022-09-10 00:10:00] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 5 次重试 
[2022-09-10 00:10:03] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 6 次重试 
[2022-09-10 00:10:06] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 7 次重试 
[2022-09-10 00:10:09] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 8 次重试 
[2022-09-10 00:10:12] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 9 次重试 
[2022-09-10 00:10:15] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 10 次重试 
[2022-09-10 00:10:18] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1662739786,"self_id":3515950251,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":137,"packet_sent":123,"packet_lost":0,"message_received":3,"message_sent":2,"last_message_time":1662738908,"disconnect_times":0,"lost_times":0}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2022-09-10 00:10:20] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 00:10:22] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 00:10:23] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 00:10:25] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 00:10:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 00:10:28] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 00:10:29] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 4 次重试 
[2022-09-10 00:10:31] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 4 次重试 
[2022-09-10 00:10:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 5 次重试 
[2022-09-10 00:10:34] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 5 次重试 
[2022-09-10 00:10:36] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 6 次重试 
[2022-09-10 00:10:37] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 6 次重试 
[2022-09-10 00:10:39] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 7 次重试 
[2022-09-10 00:10:40] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 7 次重试 
[2022-09-10 00:10:42] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 8 次重试 
[2022-09-10 00:10:43] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 8 次重试 
[2022-09-10 00:10:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 9 次重试 
[2022-09-10 00:10:47] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 9 次重试 
[2022-09-10 00:19:36] [WARNING]: 转换CQ码 [CQ:image,file=D:\projects_python\cqhttp-based-botv0539208537.png,type=show,id=40000] 时出现错误: invalid image 将原样发送. 
[2022-09-10 00:20:18] [WARNING]: 转换CQ码 [CQ:image,file=D:\projects_python\cqhttp-based-botv0539208537.png] 时出现错误: invalid image 将原样发送. 
[2022-09-10 00:27:30] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 08:54:22] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2022-09-10 08:54:35] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 08:54:38] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 08:54:41] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 08:54:44] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 4 次重试 
[2022-09-10 08:54:47] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 5 次重试 
[2022-09-10 08:54:51] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 6 次重试 
[2022-09-10 08:54:54] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 7 次重试 
[2022-09-10 08:54:57] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 8 次重试 
[2022-09-10 08:55:00] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 9 次重试 
[2022-09-10 08:55:03] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 10 次重试 
[2022-09-10 08:55:06] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1662771273,"self_id":3515950251,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":60,"packet_sent":52,"packet_lost":0,"message_received":0,"message_sent":0,"last_message_time":0,"disconnect_times":0,"lost_times":0}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2022-09-10 08:55:08] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 08:55:11] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 09:18:08] [WARNING]: 发送临时会话消息失败: 已关闭临时会话信息发送功能 
[2022-09-10 09:19:15] [WARNING]: 发送临时会话消息失败: 已关闭临时会话信息发送功能 
[2022-09-10 11:59:08] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2022-09-10 11:59:21] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 11:59:24] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 11:59:27] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 11:59:30] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 4 次重试 
[2022-09-10 11:59:33] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 5 次重试 
[2022-09-10 11:59:46] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 11:59:49] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 11:59:52] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 11:59:55] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 4 次重试 
[2022-09-10 11:59:58] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 5 次重试 
[2022-09-10 12:00:01] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 6 次重试 
[2022-09-10 12:00:04] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 7 次重试 
[2022-09-10 12:01:31] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2022-09-10 12:01:54] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 12:01:57] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 12:02:00] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 12:02:03] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 4 次重试 
[2022-09-10 12:02:06] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 5 次重试 
[2022-09-10 12:02:09] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 6 次重试 
[2022-09-10 12:02:12] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 7 次重试 
[2022-09-10 12:02:15] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 8 次重试 
[2022-09-10 12:02:18] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 9 次重试 
[2022-09-10 12:02:21] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 10 次重试 
[2022-09-10 12:02:24] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1662782512,"self_id":3515950251,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":57,"packet_sent":50,"packet_lost":0,"message_received":0,"message_sent":0,"last_message_time":0,"disconnect_times":0,"lost_times":0}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2022-09-10 12:02:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 12:02:29] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 12:02:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 13:16:01] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:16:01] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:16:01] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 540291120 
[2022-09-10 13:16:01] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 541221220 
[2022-09-10 13:16:01] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:16:01] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:16:02] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:16:02] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:16:02] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:16:02] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:16:03] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:16:03] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:16:03] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:16:03] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:16:04] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:16:04] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:16:04] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:16:04] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:16:05] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:16:05] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:16:05] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:16:05] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:16:06] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:16:06] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:16:44] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:16:44] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:16:44] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 540291120 
[2022-09-10 13:16:44] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 541221220 
[2022-09-10 13:16:44] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:16:44] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:16:45] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:16:45] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:16:45] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:16:45] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:16:46] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:16:46] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:16:47] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:16:47] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:16:47] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:16:47] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:16:48] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:16:48] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:16:48] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:16:49] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:16:50] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:16:50] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:16:50] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:16:50] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:16:51] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:16:51] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:17:20] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:17:20] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:17:20] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 540291120 
[2022-09-10 13:17:20] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 541221220 
[2022-09-10 13:17:20] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:17:21] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:17:21] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:17:21] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:17:21] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:17:22] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:17:22] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:17:23] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:17:23] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:17:23] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:17:23] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:17:24] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:17:25] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:17:26] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:17:26] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:17:26] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:17:26] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:19:41] [WARNING]: 虚拟设备信息不存在, 将自动生成随机设备. 
[2022-09-10 13:20:26] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:20:26] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:20:26] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 540291120 
[2022-09-10 13:20:26] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 541221220 
[2022-09-10 13:20:27] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:20:27] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:20:27] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:20:27] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:20:28] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:20:28] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:20:28] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:20:28] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:20:29] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:20:29] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:20:30] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:20:30] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:20:30] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:20:30] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:20:31] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:20:31] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:20:31] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:20:31] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:20:32] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:20:32] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:20:32] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:20:32] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:20:33] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:20:33] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:20:33] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:20:33] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:20:34] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:20:34] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:20:34] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:20:34] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:20:35] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:20:35] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:20:35] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:20:35] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:20:36] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:20:36] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:20:36] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:20:36] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:20:37] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:20:37] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:20:37] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:20:37] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:20:38] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:20:38] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:20:38] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:20:38] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:20:39] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:20:41] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:20:42] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:20:42] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:15] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:15] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:15] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 540291120 
[2022-09-10 13:23:15] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 541221220 
[2022-09-10 13:23:15] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:15] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:16] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:16] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:16] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:16] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:17] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:17] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:17] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:17] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:18] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:18] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:18] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:18] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:19] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:19] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:19] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:19] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:20] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:20] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:20] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:20] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:21] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:21] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:21] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:21] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:22] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:22] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:22] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:22] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:23] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:23] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:23] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:23] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:24] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:24] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:24] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:24] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:25] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:25] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:25] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:25] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:26] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:26] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:26] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:26] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:27] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:27] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:27] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:27] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:28] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:28] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:28] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:28] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:29] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:29] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:30] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:30] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:30] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:30] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:31] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:31] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:31] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:31] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:32] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:32] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:32] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:32] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:33] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:33] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:33] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:33] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:34] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:34] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:34] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:34] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:35] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:35] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:35] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:36] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:23:37] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:23:37] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:24:29] [WARNING]: 虚拟设备信息不存在, 将自动生成随机设备. 
[2022-09-10 13:24:36] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:24:36] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:24:36] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 540291120 
[2022-09-10 13:24:36] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 541221220 
[2022-09-10 13:24:37] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:24:37] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:24:37] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:24:37] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:24:38] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:24:38] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:24:38] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:24:38] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:24:39] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:24:39] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:24:39] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:24:39] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:24:40] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:24:40] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:24:40] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:24:41] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:24:41] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:24:42] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:24:42] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:24:42] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:24:42] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:24:43] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:24:43] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:24:43] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 1213486160 
[2022-09-10 13:24:44] [ERROR]: Protocol -> parse incoming packet error: invalid packet length 791752241 
[2022-09-10 13:25:33] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2022-09-10 13:25:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 13:25:48] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 13:25:51] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 13:25:54] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 4 次重试 
[2022-09-10 13:25:57] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 5 次重试 
[2022-09-10 13:26:00] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 6 次重试 
[2022-09-10 13:26:03] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 7 次重试 
[2022-09-10 13:26:06] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 8 次重试 
[2022-09-10 13:26:20] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 13:26:23] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 13:26:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 13:26:29] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 4 次重试 
[2022-09-10 13:26:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 5 次重试 
[2022-09-10 13:26:35] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 6 次重试 
[2022-09-10 13:26:50] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 13:26:53] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 13:26:56] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 13:26:59] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 4 次重试 
[2022-09-10 13:27:02] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 5 次重试 
[2022-09-10 13:27:05] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 6 次重试 
[2022-09-10 18:19:09] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2022-09-10 18:19:22] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 18:19:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 18:19:29] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 18:19:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 4 次重试 
[2022-09-10 18:19:35] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 5 次重试 
[2022-09-10 18:19:38] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 6 次重试 
[2022-09-10 18:19:41] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 7 次重试 
[2022-09-10 18:19:44] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 8 次重试 
[2022-09-10 18:19:47] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 9 次重试 
[2022-09-10 18:19:50] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 10 次重试 
[2022-09-10 18:19:53] [WARNING]: 上报 Event 数据 {"post_type":"meta_event","meta_event_type":"heartbeat","time":1662805160,"self_id":3515950251,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":61,"packet_sent":53,"packet_lost":0,"message_received":0,"message_sent":0,"last_message_time":0,"disconnect_times":0,"lost_times":0}},"interval":5000}
 到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上限 
[2022-09-10 18:39:59] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2022-09-10 18:40:13] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 18:40:16] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 18:40:19] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 18:40:22] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 4 次重试 
[2022-09-10 18:40:25] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 18:40:25] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 5 次重试 
[2022-09-10 18:40:28] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 18:40:28] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 6 次重试 
[2022-09-10 18:40:31] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 18:40:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 7 次重试 
[2022-09-10 18:40:34] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 4 次重试 
[2022-09-10 18:40:35] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 8 次重试 
[2022-09-10 18:40:37] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 5 次重试 
[2022-09-10 18:40:38] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 9 次重试 
[2022-09-10 18:40:40] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 6 次重试 
[2022-09-10 18:40:41] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 10 次重试 
[2022-09-10 18:47:29] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2022-09-10 18:58:36] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2022-09-10 19:03:23] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 19:03:24] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 19:05:37] [WARNING]: Protocol -> device lock is disable. http api may fail. 
[2022-09-10 19:05:51] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 19:05:51] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 19:05:54] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-09-10 19:05:54] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 19:05:54] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 19:05:57] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-09-10 19:05:57] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 19:05:57] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 19:06:00] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-09-10 19:06:00] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 4 次重试 
[2022-09-10 19:06:00] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 4 次重试 
[2022-09-10 19:06:03] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 4 次重试 
[2022-09-10 19:06:03] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 5 次重试 
[2022-09-10 19:06:03] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 5 次重试 
[2022-09-10 19:06:06] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 5 次重试 
[2022-09-10 19:06:06] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 6 次重试 
[2022-09-10 19:06:06] [WARNING]: 上报 Event 数据到 http://127.0.0.1:5701/ 失败: Post "http://127.0.0.1:5701/": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将进行第 6 次重试 
